devtools/composer.json

```diff
@@ -4,7 +4,7 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "apigen/apigen": "^2.8",
-        "fabpot/php-cs-fixer": "^1.11",
+        "friendsofphp/php-cs-fixer": "^1.11,<1.11.5",
```

Comment:
A am totally for splitting work for multiple PRs, no need to do everything at once.
Exactly for that I commented composer update while working on Travis matrix change ;)